_posts/en/newsletters/2023-12-06-newsletter.md

```diff
@@ -0,0 +1,179 @@
+      vbytes.  For example:
+
+      ![Incomparable chunkings](/img/posts/2023-12-incomparable-chunkings.png)
+
+      As one of the theorems in the previously linked thread notes, "if
+      one has two incomparable chunkings for a graph, then another
+      chunking must exist which is strictly better than both".  That
+      means an effective method for merging two different incomparable
+      chunkings can be a powerful tool for improving miner
+      profitability.  For example, miners Alice and Bob could both try
+      to find the optimal chunking and could share their results with
+      each other in a way that would guarantee they'd both obtain a
+      third chunking that was at least as good as either of their
+      original chunkings---and possibly better.
+
+    - [Post-cluster package RBF][cluster rbf] discusses an alternative
+      to the rules currently used for [replace by fee][topic rbf], which
+      are known to cause multiple problems, including [pinning
+      attacks][topic transaction pinning].  The goal is to accept any
+      replacement transaction or [package of transactions][topic package
+      relay] that will produce a chunking that is better than an
+      equivalent chunking of the current mempool (as long as the
+      replacement also follows normal anti-DoS relay policy, such as
+      having the replacement pay for its own relay by the _incremental
+      relay feerate_, which is currently 1 sat per vbyte).  Several
+      [other][cluster rbf-old1] threads [also][cluster rbf-old2]
+      discussed [this][cluster rbf-old3] topic.
+
+- **Testing with warnet:** Matthew Zipkin [posted][zipkin warnet] to
+  Delving Bitcoin with the results of some simulations he's run using
+  [warnet][], a program that launches a large number of Bitcoin nodes
+  with a defined set of connections between them (usually on a test
+  network).  Zipkin results show the extra memory that would be used if
+  several proposed changes to peer management code are merged (either
+  independently or together).  He also notes that he's excited to use
+  simulations for testing other proposed changes and to quantify the
+  effect of proposed attacks.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+FIXME:LarryRuane
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/28368#l-30FIXME"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.0rc3][] is a release candidate for the next major
+  version of the predominant full-node implementation. There's a [testing
```

Comment:
Per https://github.com/bitcoinops/bitcoinops.github.io/pull/1400#discussion_r1402368137:
```suggestion
  version of the predominant full node implementation. There's a [testing
```

Reply:
It reads better to me with the hyphen, because it's conceptually a single, unitary thing (rather than saying it's a node that happens to be "full" whatever that means -- is it out of disk space?), but I don't have a strong opinion.

Reply:
I'm not knowledgeble enough to discuss the language perspective, I'm just saying this goes against long established terminology both at Bitcoin Optech and most everywhere else I could find. The hyphenated version was never used on Optech until a few weeks ago. I'd think we'd like to keep terminology consistent since we even have a style guide for this very purpose.

Reply:
@LarryRuane would you hyphenate "Bitcoin node implementation" and "LN node implementation"?

Given that Optech frequently discusses things like peer networks that can be thought of topologically as represented by nodes and edges, we modify the noun "node" a lot to make it clear what type of node we're talking about, so we probably want a general policy.